human_robot_collaboration/src/baxter_display.cpp

```diff
@@ -8,11 +8,13 @@
 #include <sensor_msgs/image_encodings.h>
 
 #include "human_robot_collaboration_msgs/ArmState.h"
+#include "ros_speech2text/event.h"
 
 using namespace std;
 using namespace human_robot_collaboration_msgs;
 
 #define DEFAULT_DURATION 10.0  // [s]
+#define LISTENING 1
```

Comment:
do you need this? It seems an overkill to define a variable that is true when is true :sunglasses: